lib/src/collection/multimap.dart

```diff
@@ -187,8 +188,8 @@ abstract class _BaseMultimap<K, V, C extends Iterable<V>>
   }
 
   @override
-  void removeWhere(bool predicate(K key, V value)) {
-    final emptyKeys = Set<K>();
+  void removeWhere(bool Function(K key, V value) predicate) {
+    final emptyKeys = <K>{};
```

Comment:
Does `Set()` and `{}` constructors synonymous? I recall having to call `HashMap` constructor directly in Flutter code instead of `{}` because the latter used to go for `LinkeHashMap` instead. This could break downstream code that relies on key order.

Reply:
`Set` and `{}` are the same! Both save key order.